### PR TITLE
Migration of canvas test helpers

### PR DESCRIPTION
**Pull request by spydon**
This is the migration of the canvas test helpers that used to live in Flame, but was decided to be taken out and put in its own package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Canvas test helpers
+
+[MockCanvas] is a utility class for writing tests. It supports the same API
```
Comment (renancaraujo) [inline README.md]:
```suggestion
[MockCanvas] is a utility class for writing tests for canvas operations. It supports the same API
```

**Review by renancaraujo — APPROVED** [review]

File: lib/canvas_test_helpers.dart [hunk 2]
```diff
@@ -0,0 +1,7 @@
+library canvas_test_helpers;
+
+/// A Calculator.
+class Calculator {
+  /// Returns [value] plus 1.
+  int addOne(int value) => value + 1;
```
Comment (erickzanardo) [inline lib/canvas_test_helpers.dart]: We can remove this file

**Review by erickzanardo — APPROVED** [review]
One file needing removal, but other than that, LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# CHANGELOG
+
+## 0.0.1
```
Comment (wolfenrain) [inline CHANGELOG.md]: Its a "new feature" so imho it should be 0.1.0

**Review by wolfenrain — APPROVED** [review]

File: lib/canvas_test_helpers.dart (hunk 2 above)
Comment (spydon) [reply lib/canvas_test_helpers.dart]: Ah, I was going to put the exports in here, forgot about that.

File: CHANGELOG.md (hunk 3 above)
Comment (spydon) [reply CHANGELOG.md]: It was `null.null.null` before this release, so we even bumped major ;)

Comment (wolfenrain) [reply CHANGELOG.md]: I am also fine with that if this is already our defining API ;-)

**Pull request merged**